From ca7df000f5ff8e017ffb8990c79d4ee62b6fab02 Mon Sep 17 00:00:00 2001
From: joe9 <joe9mail@gmail.com>
Date: Tue, 8 Sep 2015 17:43:17 -0400
Subject: [PATCH] changed font to dejavu sans mono

---
 config.def.h | 36 +++++++++++++++++++++++++++++++++++-
 1 file changed, 35 insertions(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 52bf3a4..699869e 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,8 +5,41 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+/*
+  use the below utility from ft2demos to figure out what makes
+  the font look better
+  ftview -r 96 10 /usr/share/fonts/inconsolata_dz/Inconsolata-dz.otf
+  hintstyle = outline hinting
+
+  from http://www.freetype.org/freetype2/docs/glyphs/glyphs-2.html
+  pixel_size = point_size * resolution / 72
+  As the size of a pixel is very device dependent, size is specified
+  in points.
+  Points are a simple physical unit, where 1 point = 1/72th of an inch
+  for 10 point font, pixel size = (10 * 96)/72 ~ 13.3
+  with a smaller pixel size, some monospace system font is being used
+  instead of the specified font. watch out for this behaviour.
+
+  from https://bbs.archlinux.org/viewtopic.php?pid=555485
+  For crisp fonts make sure you dont set the font size in pixels, like so…
+  URxvt.font: xft:DejaVu Sans Mono:pixelsize=10
+  Instead do it like this…
+  URxvt.font: xft:DejaVu Sans Mono-7
+  It is subtle but I have found it always gives clearer results.
+*/
+/* autohinter (autohint) is not compatible with subpixel rendering (rgba) */
+/* -<point> is not working with the below font, hence, using pixelsize */
+/* static char font[] = "Anonymous Pro:pixelsize=15:rgba=rgb:hinting=true:antialias=true:hintstyle=hintfull:lcdfilter=lcdlight:autohint=false"; */
+/* static char font[] = "DejaVu Sans Mono-9:rgba=rgb:hinting=true:antialias=false:hintstyle=hintfull:lcdfilter=lcddefault:autohint=false"; */
+/* static char font[] = "Liberation Mono:pixelsize=12:antialias=false:autohint=false"; */
+/* static char font[] = "Liberation Mono:pixelsize=12:rgba=rgb:hinting=true:antialias=true:hintstyle=hintfull:lcdfilter=lcddefault:autohint=false"; */
+/* -<point> is not working with the below font, hence, using pixelsize */
+/* pixelsize=14 works fine too, but 15 is easier on the eye */
+/* static char font[] = "Inconsolata:pixelsize=15:hinting=true:antialias=true:hintstyle=hintfull:lcdfilter=lcddefault"; */
+/* static char font[] = "Terminus"; */
+/* static char font[] = "Liberation Mono:pixelsize=12:antialias=false:autohint=false"; */
+/* static char font[] = "DejaVu Sans Mono:pixelsize=12:antialias=false:autohint=false"; */
+static char font[] = "DejaVu Sans Mono:pixelsize=12:";
 static int borderpx = 2;
 
 /*
--
2.4.6

