From 05ac0640eab5b1fe04ea1a8ef005f324ac695b00 Mon Sep 17 00:00:00 2001
From: joe9 <joe9mail@gmail.com>
Date: Tue, 8 Sep 2015 17:04:47 -0400
Subject: [PATCH 2/7] changed word delimiters

---
 config.def.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 3a87709..82acab9 100644
--- a/config.def.h
+++ b/config.def.h
@@ -66,7 +66,8 @@ static float chscale = 1.0;
  *
  * More advanced example: " `'\"()[]{}"
  */
-static char worddelimiters[] = " ";
+/* static char worddelimiters[] = " "; */
+static char worddelimiters[] = " `'\"()[]{}";
 
 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
-- 
2.4.6

