From 7738ed2c057899ae0164a5d90cc24f4cba44836f Mon Sep 17 00:00:00 2001
From: joe9 <joe9mail@gmail.com>
Date: Mon, 18 Feb 2013 12:59:42 -0500
Subject: [PATCH 2/2] checkout unknown private set/reset mode 1005

---
 st.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/st.c b/st.c
index 8a48571..2135b8e 100644
--- a/st.c
+++ b/st.c
@@ -1586,8 +1586,8 @@ tsetmode(bool priv, bool set, int *args, int narg) {
 				break;
 			default:
 				fprintf(stderr,
-					"erresc: unknown private set/reset mode %d\n",
-					*args);
+					"erresc: unknown private set/reset mode %d, set: %d\n",
+					*args,set);
 				break;
 			}
 		} else {
--
1.7.12.4

