From e1704a22b5d3238bfe8ecfe95a6113a5e511090b Mon Sep 17 00:00:00 2001
From: joe9 <joe9mail@gmail.com>
Date: Sun, 11 Oct 2015 22:20:10 -0400
Subject: [PATCH] changed default colors

---
 config.def.h | 99 ++++++++++++++++++++++++++++++++++++++++++------------------
 1 file changed, 69 insertions(+), 30 deletions(-)

diff --git a/config.def.h b/config.def.h
index 860ce3d..a0840e7 100644
--- a/config.def.h
+++ b/config.def.h
@@ -68,32 +68,66 @@ static char termname[] = "st-256color";
 static unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
+/* static const char *colorname[] = { */
+/*	/\* 8 normal colors *\/ */
+/*	"black", */
+/*	"red3", */
+/*	"green3", */
+/*	"yellow3", */
+/*	"blue2", */
+/*	"magenta3", */
+/*	"cyan3", */
+/*	"gray90", */
+
+/*	/\* 8 bright colors *\/ */
+/*	"gray50", */
+/*	"red", */
+/*	"green", */
+/*	"yellow", */
+/*	"#5c5cff", */
+/*	"magenta", */
+/*	"cyan", */
+/*	"white", */
+
+/*	[255] = 0, */
+
+/*	/\* more colors can be added after 255 to use with DefaultXX *\/ */
+/*	"#cccccc", */
+/*	"#555555", */
+/* }; */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+   /* 8 normal colors */
+   "#2e3436",
+   "#cc0000",
+   "#4e9a06",
+   "#c4a000",
+   "#00afff",
+   "#75507b",
+   "#0b939b",
+   "#d3d7cf",
+
+   /* 8 bright colors */
+   "#555753",
+   "#ef2929",
+   "#8ae234",
+   "#fce94f",
+   "#729fcf",
+   "#ad7fa8",
+   "#00f5e9",
+   "#eeeeec",
+
+   [255] = 0,
+
+   /* more colors can be added after 255 to use with DefaultXX */
+   "#cccccc",
+   "#555555",
+   "#333333",
+   /* URxvt*foreground : DarkGrey*/
+   "#cacaca",
+   /* URxvt*background : tinted black */
+   "#121212",
+   /* URxvt.cursorColor:          #EB122E */
+   "#EB122E",
 };
 
 
@@ -101,9 +135,12 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+/* static unsigned int defaultfg = 7; */
+/* static unsigned int defaultbg = 0; */
+/* static unsigned int defaultcs = 256; */
+static unsigned int defaultfg = 259;
+static unsigned int defaultbg = 260;
+static unsigned int defaultcs = 261;
 static unsigned int defaultrcs = 257;
 
 /*
@@ -127,8 +164,10 @@ static unsigned int mousebg = 0;
  * will reverse too. Another logic would only make the simple feature too
  * complex.
  */
-static unsigned int defaultitalic = 11;
-static unsigned int defaultunderline = 7;
+/* static unsigned int defaultitalic = 11; */
+/* static unsigned int defaultunderline = 7; */
+static unsigned int defaultitalic = 259;
+static unsigned int defaultunderline = 259;
 
 /*
  * Internal mouse shortcuts.
-- 
2.4.9

