diff --git a/st.c b/st.c
index 040638a..67d78eb 100644
--- a/st.c
+++ b/st.c
@@ -1885,11 +1885,13 @@ tsetmode(bool priv, bool set, int *args, int narg) {
 				MODBIT(term.mode, set, MODE_BRCKTPASTE);
 				break;
 			/* Not implemented mouse modes. See comments there. */
-			case 1001: /* mouse highlight mode; can hang the
-				      terminal by design when implemented. */
-			case 1005: /* UTF-8 mouse mode; will confuse
+			/* joe9 changed the below line to ignore these
+			 * error lines in the logs. */
+			case 1005: break; /* UTF-8 mouse mode; will confuse
 				      applications not supporting UTF-8
 				      and luit. */
+			case 1001: /* mouse highlight mode; can hang the
+				      terminal by design when implemented. */
 			case 1015: /* urxvt mangled mouse mode; incompatible
 				      and can be mistaken for other control
 				      codes. */
