--- Makefile.original	2014-05-02 11:36:38.198084573 -0500
+++ Makefile	2014-05-02 11:58:49.695069861 -0500
@@ -1,5 +1,5 @@
 #
-# Makefile for proxychains (requires GNU make), stolen from musl
+# Makefile for proxychains-ng (requires GNU make), stolen from musl
 #
 # Use config.mak to override any of the following variables.
 # Do not make changes here.
@@ -16,14 +16,14 @@
 SRCS = $(sort $(wildcard src/*.c))
 OBJS = $(SRCS:.c=.o)
 LOBJS = src/nameinfo.o \
-        src/core.o src/common.o src/libproxychains.o src/shm.o \
+        src/core.o src/common.o src/libproxychains-ng.o src/shm.o \
         src/allocator_thread.o src/ip_type.o src/stringdump.o \
         src/hostentdb.o src/hash.o src/debug.o
 
-CFLAGS  += -Wall -O0 -g -std=c99 -D_GNU_SOURCE -pipe
+CFLAGS  += -Wall -std=c99 -D_GNU_SOURCE
 NO_AS_NEEDED = -Wl,--no-as-needed
 LIBDL   = -ldl
-LDFLAGS = -shared -fPIC $(NO_AS_NEEDED) $(LIBDL) -lpthread
+LIBS    = -lpthread -ldl
 INC     = 
 PIC     = -fPIC
 AR      = $(CROSS_COMPILE)ar
@@ -33,11 +33,11 @@
 LD_SET_SONAME = -Wl,-soname=
 INSTALL_FLAGS = -D -m
 
-LDSO_PATHNAME = libproxychains4.$(LDSO_SUFFIX)
+LDSO_PATHNAME = libproxychains-ng.$(LDSO_SUFFIX)
 
 SHARED_LIBS = $(LDSO_PATHNAME)
 ALL_LIBS = $(SHARED_LIBS)
-PXCHAINS = proxychains4
+PXCHAINS = proxychains-ng
 ALL_TOOLS = $(PXCHAINS)
 
 -include config.mak
@@ -50,7 +50,7 @@
 
 install-config:
 	install -d $(DESTDIR)$(sysconfdir)
-	install $(INSTALL_FLAGS) 644 src/proxychains.conf $(DESTDIR)$(sysconfdir)/
+	install $(INSTALL_FLAGS) 644 src/proxychains-ng.conf $(DESTDIR)$(sysconfdir)/
 
 install: 
 	install -d $(DESTDIR)$(bindir)/ $(DESTDIR)$(libdir)/
@@ -66,10 +66,10 @@
 	$(CC) $(CFLAGS) $(CFLAGS_MAIN) $(INC) $(PIC) -c -o $@ $<
 
 $(LDSO_PATHNAME): $(LOBJS)
-	$(CC) $(LDFLAGS) $(LD_SET_SONAME)$(LDSO_PATHNAME) -o $@ $(LOBJS)
+	$(CC) $(LDFLAGS) -shared $(LD_SET_SONAME)$(LDSO_PATHNAME) -o $@ $(LOBJS)
 
 $(ALL_TOOLS): $(OBJS)
-	$(CC) src/main.o src/common.o -o $(PXCHAINS)
+	$(CC) $(LDFLAGS) src/main.o src/common.o -o $(PXCHAINS)
 
 
 .PHONY: all clean install install-config
